### ISSUE TITLE
No intellisense for Azure SQL Database

### ISSUE THREAD

**Issue by Brian-Ding** [microsoft/azuredatastudio#15557]
- Azure Data Studio Version:
Version: 1.28.0 (user setup)
Commit: 13e362762762e5cb76a5c8afd2bc650f48c2d3d8
Date: 2021-04-15T00:24:15.710Z
VS Code: 1.51.0
Electron: 9.4.3
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Windows_NT x64 10.0.19041

Steps to Reproduce:

1. Open Azure Studio
2. Connect to Azure SQL Database
3. After typing "SELECT * FROM ", no intellisense for table names, after typing "WHERE ", no intellisense for column names
4. Connect to local machine sql server or on-premises, intellisense for table/column names works correctly

**Comment by alanrenmsft** [comment]
@Brian-Ding did you add schema name to the table? I just tried `select * from sys.` and I am able to see the table names.

**Comment by Brian-Ding** [comment]
> @Brian-Ding did you add schema name to the table? I just tried `select * from sys.` and I am able to see the table names.

Yes I tried, the intellisense applies to master Database, but not the one I created
